### ISSUE TITLE
Community plugins are not installed in the docker image

### ISSUE THREAD

**Issue by dummys** [sk4la/volatility3-docker#26]
Hello,
In the documentation you wrote that you install all the community3 plugins, which is wrong for latest plugins.
In fact you only download the pdf for documentation in each folder:
``bash
~/workspace $ tree /usr/local/share/volatility3/plugins/community3/
/usr/local/share/volatility3/plugins/community3/
├── Block_PTE_Malfind
│   └── README.md
├── Breen_LastPass_ISF_Rich_CS
│   └── README.md
├── Chechik_Weber_CheckSpoof
│   ├── Or Chechik and Inon Weber - Volatility contest 2020 submission.pdf
│   └── README.md
├── Guyard_VolWeb
│   └── README.md
├── Team_BTC_Cryptoscan
│   ├── README.md
│   └── The Extraction of Cryptocurrency Transaction Information in Memory_CryptoScan.pdf
├── Vergeer_OpenSSH
│   ├── Decrypting OpenSSH sessions for fun and profit - vol.pptx
│   └── README.md
└── Zohar_VolExp
    ├── Plugins documontation.pdf
    └── README.md
```
Maybe we could download it from github directly ?

**Comment by sk4la** [comment]
Indeed. The old Volatility 2 community plugins repository used to host the original versions of many plugins. Unfortunately, that's no longer the case with Volatility 3.

I'm a bit reluctant to include third-party code that lives outside the official Volatility repositories... If you'd like to use community plugins that aren't part of the main [`volatilityfoundation/community3`](https://github.com/volatilityfoundation/community3) repository, you can customize the `sk4la/volatility3` image accordingly. See this [example](https://github.com/sk4la/volatility3-docker/blob/master/README.md#example-4-overloading-the-docker-images-to-fit-your-needs) for guidance on how to overload the Docker image to suit your needs.

I've updated the README to make this clearer.
